src/commands/logstream/stopStreamingLogs.ts

```diff
@@ -0,0 +1,19 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzureTreeDataProvider, IAzureNode } from 'vscode-azureextensionui';
+import { FunctionAppTreeItem } from '../../tree/FunctionAppTreeItem';
+import { ILogStreamTreeItem } from './ILogStreamTreeItem';
+
+export async function stopStreamingLogs(tree: AzureTreeDataProvider, node?: IAzureNode<ILogStreamTreeItem>): Promise<void> {
```

Comment:
Looks like this function (and corresponding context menu item) would be visible for every Function App. Is there a way to filter the node picker and context menu presence down to Apps that have a logStream present? (I see that being done with the `node.treeItem.logStream` conditional - I think the UX might be cleaner though)

Reply:
It's not recommended to hide things like context-menu commands based on state. Users will get confused about why it displays sometimes and not other times. Best practice is actually to gray-out the option, but I don't think that's possible in VS Code

In any case, I'll just display a warning to the user in this case.

Reply:
Yeah, greying out an option makes waay more sense. I stand corrected. :)